clouddriver-openstack/src/main/resources/asg.yaml

```diff
@@ -44,28 +80,52 @@ resources:
           flavor: {get_param: flavor}
           image: {get_param: image}
           internal_port: {get_param: internal_port}
-          # TODO this is here for demo purposes - will need to figure if this shoudl be parameterized
+          # This is used by ceilometer to autoscale against instances that are part of this stack
           metadata: {"metering.stack": {get_param: "OS::stack_id"}}
           network_id: {get_param: network_id}
           pool_id: {get_param: pool_id}
           security_groups: {get_param: security_groups}
           subnet_id: {get_param: subnet_id}
-  # TODO parameterize
   web_server_scaleup_policy:
     type: OS::Heat::ScalingPolicy
     properties:
       adjustment_type: change_in_capacity
       auto_scaling_group_id: {get_resource: asg}
-      cooldown: 60
-      scaling_adjustment: 1
-  # TODO parameterize
+      cooldown: {get_param: scaleup_cooldown}
+      scaling_adjustment: {get_param: scaleup_adjustment}
   web_server_scaledown_policy:
     type: OS::Heat::ScalingPolicy
     properties:
       adjustment_type: change_in_capacity
       auto_scaling_group_id: {get_resource: asg}
-      cooldown: 60
-      scaling_adjustment: -1
+      cooldown: {get_param: scaledown_cooldown}
+      scaling_adjustment: {get_param: scaledown_adjustment}
+  cpu_alarm_high:
```

Comment:
This is called cpu_alarm but the meter name could be network related. Does that difference matter?